### PR TITLE
Evaluate

### PR DESCRIPTION
**Pull request by SebWae**
Script to evaluate our models. Returns a dictionary of evaluation metrics incl. 
- Accuracy
- Macro and class-weighted F1 score
- Confusion matrix

Is currently only configured to evaluate the `BaseANN` model.

### CODE REVIEW

**Comment by krusand** [comment]
Looks good to me.
Let's increase the batch size of the DataLoader, I think that is also why pytorch-lightning was so slow. Good catch.
I will accept and merge now.
